proofs/cbmc/crypto_kem_dec/Makefile

```diff
@@ -20,7 +20,7 @@ PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
 PROJECT_SOURCES += $(SRCDIR)/mlkem/src/kem.c
 
 CHECK_FUNCTION_CONTRACTS=mlk_dec
-USE_FUNCTION_CONTRACTS=mlk_sha3_512 mlk_sha3_256 mlk_indcpa_enc mlk_indcpa_dec mlk_shake256 mlk_ct_memcmp mlk_ct_cmov_zero memcmp mlk_zeroize
+USE_FUNCTION_CONTRACTS=mlk_check_sk mlk_sha3_512 mlk_sha3_256 mlk_indcpa_enc mlk_indcpa_dec mlk_shake256 mlk_ct_memcmp mlk_ct_cmov_zero memcmp mlk_zeroize
```

Comment:
Something is off here. `crypto_kem_dec` does not use `memcmp` - doesn't CBMC usually fail when a function in `USE_FUNCTION_CONTRACTS` isn't actually called, @rod-chapman?

Regardless, `memcmp` should re removed here.

Reply:
I am happy now. memcmp removed and all proofs OK.